Runtime Error

When I try to process [https://github.com/willfarrell/Browsers](https://github.com/willfarrell/Browsers) I get the following error:
```
panic: runtime error: index out of range

goroutine 1 [running]:
gopkg.in/src-d/enry%2ev1.getInterpreter(0xc42198d080, 0x81, 0x281, 0x12, 0xbec0a0)
        /tmp/enry-dfekhdox/src/gopkg.in/src-d/enry.v1/common.go:289 +0x396
gopkg.in/src-d/enry%2ev1.GetLanguagesByShebang(0xc4200ea28a, 0x12, 0xc42198d080, 0x81, 0x281, 0xbeb730, 0x0, 0x0, 0x0, 0x0, ...)
        /tmp/enry-dfekhdox/src/gopkg.in/src-d/enry.v1/common.go:270 +0x43
gopkg.in/src-d/enry%2ev1.GetLanguages(0xc4200ea28a, 0x12, 0xc42198d080, 0x81, 0x281, 0x0, 0x0, 0xc420e3cd60)
        /tmp/enry-dfekhdox/src/gopkg.in/src-d/enry.v1/common.go:126 +0x127
gopkg.in/src-d/enry%2ev1.GetLanguage(0xc4200ea28a, 0x12, 0xc42198d080, 0x81, 0x281, 0x0, 0x0)
        /tmp/enry-dfekhdox/src/gopkg.in/src-d/enry.v1/common.go:38 +0x53
main.main.func1(0xc4200ea230, 0x6c, 0xbbd740, 0xc421d17a00, 0x0, 0x0, 0x0, 0x0)
        /tmp/enry-dfekhdox/src/gopkg.in/src-d/enry.v1/cli/enry/main.go:80 +0x664
path/filepath.walk(0xc4200ea230, 0x6c, 0xbbd740, 0xc421d17a00, 0xc4209915f0, 0x0, 0x0)
        /usr/lib/go-1.8/src/path/filepath/path.go:351 +0x81
path/filepath.walk(0xc4215483c0, 0x59, 0xbbd740, 0xc421d17930, 0xc4209915f0, 0x0, 0x0)
        /usr/lib/go-1.8/src/path/filepath/path.go:376 +0x414
path/filepath.walk(0xc4215482a0, 0x53, 0xbbd740, 0xc421d17860, 0xc4209915f0, 0x0, 0x0)
        /usr/lib/go-1.8/src/path/filepath/path.go:376 +0x414
path/filepath.walk(0xc420bd5d60, 0x4a, 0xbbd740, 0xc421d17790, 0xc4209915f0, 0x0, 0x0)
        /usr/lib/go-1.8/src/path/filepath/path.go:376 +0x414
path/filepath.walk(0xc420173680, 0x33, 0xbbd740, 0xc420954a90, 0xc4209915f0, 0x0, 0x30)
        /usr/lib/go-1.8/src/path/filepath/path.go:376 +0x414
path/filepath.Walk(0xc420173680, 0x33, 0xc4209915f0, 0x0, 0xc4209915c0)
        /usr/lib/go-1.8/src/path/filepath/path.go:398 +0x14c
main.main()
```
Here is the full log:
[stderr.txt](https://github.com/src-d/enry/files/1170122/stderr.txt)

Update: 

Minimal reproducible example:
```
wget https://raw.githubusercontent.com/mjirik/lisa/master/tests/ubuntu_installer_test.sh
enry ubuntu_installer_test.sh

panic: runtime error: index out of range
```

---

with latest build of enry
```
cd ~$GOPATH/src/gopkg.in/src-d/
rm -rf enry.v1
git clone https://github.com/src-d/enry.git enry.v1
make build-cli
cp enry ~$GOPATH/bin

enry -h                                                                                                    
  enry v1.3.2 build: 07-25-2017_09_24_59 commit: 7cd8904, based on linguist commit: 37979b2
  enry, A simple (and faster) implementation of github/linguist
```

Narrow it down to 4 files from `lisa` repository above:
```
find . -type f -not -path "./.git/*" | xargs -I{} enry {} 2>&1 |  grep -c "panic: runtime error: index out of range"
```

https://github.com/mjirik/lisa/blob/master/applications/LISA.app/Contents/MacOS/lisa
https://github.com/mjirik/lisa/blob/master/installer.sh
https://github.com/mjirik/lisa/blob/master/lisa.sh
https://github.com/mjirik/lisa/blob/master/tests/ubuntu_installer_test.sh

--- a/common.go
+++ b/common.go
@@ -284,14 +284,16 @@ func getInterpreter(data []byte) (interpreter string) {
 
 	// skip shebang
 	line = bytes.TrimSpace(line[2:])
-
 	splitted := bytes.Fields(line)
+	if len(splitted) == 0 {
+		return ""
+	}
+
 	if bytes.Contains(splitted[0], []byte("env")) {
 		if len(splitted) > 1 {
 			interpreter = string(splitted[1])
 		}
 	} else {
-
 		splittedPath := bytes.Split(splitted[0], []byte{'/'})
 		interpreter = string(splittedPath[len(splittedPath)-1])
 	}

--- a/common_test.go
+++ b/common_test.go
@@ -212,6 +212,7 @@ println("The shell script says ",vm.arglist.concat(" "));`
 		{name: "TestGetLanguagesByShebang_8", content: []byte(`#!bash`), expected: []string{"Shell"}},
 		{name: "TestGetLanguagesByShebang_9", content: []byte(multilineExecHack), expected: []string{"Tcl"}},
 		{name: "TestGetLanguagesByShebang_10", content: []byte(multilineNoExecHack), expected: []string{"Shell"}},
+		{name: "TestGetLanguagesByShebang_11", content: []byte(`#!`), expected: nil},
 	}
 
 	for _, test := range tests {

